src/App.js

```diff
@@ -1,15 +1,25 @@
 import React, { Component } from 'react';
-import './App.css';
 import SearchBar from './Components/SearchBar';
-
-const starWarsApiUrl ='https://swapi.co/api/people/';
+import SearchBarDetails from './Components/SearchBarDetails';
+import * as Constants from './Constants/Constants';
+import './App.css';
 
 class App extends Component {
 
   constructor(){
     super()
     this.state = {
-      searchedItem: { text: '', key: '' }
+      searchedItem: { text: '', key: '' },
+      foundDetails: null,
+      searchedItemDetails:{ 
+        text: {
+          name:'',
+          height:'',
+          mass:'',
+          birthday:'',
+          gender:''
+        },
+        key: '' }
     }
     this.handleSearchInput.bind(this);
     this.addItemForSearch.bind(this);
```

Comment:
You don't need to ```bind``` here, those functions are already arrow functions, so they take up their parent's ```this```.